### ISSUE TITLE
tox 3.8 update breaks this package

### ISSUE THREAD

**Issue by baxtersa** [pglass/tox-pip-version#7]
Hey there!

I started getting the following error when using this package with tox 3.8 (released yesterday).

```
AttributeError: 'Action' object has no attribute 'venvname'
```

from this line https://github.com/pglass/tox-pip-version/blob/bf1be426fcf623921bd1c4017de0c5320b96c6c3/tox_pip_version/hooks.py#L37

Looks like the tox update might have changed the name of some things. Thought I'd let you know!

**Comment by baxtersa** [comment]
Following up, it looks like it was renamed to `Action.name`

**Comment by pglass** [comment]
I see, thanks! Should be straightforward to fix in a backwards-compatible way.

I should add a note to the readme, too, that this relies on one of tox's [unstable plugin interfaces](https://tox.readthedocs.io/en/latest/plugins.html#tox.hookspecs.tox_testenv_create).

**Comment by drewbrew** [comment]
I merged Paul's fix. Thanks for reporting!

**Comment by baxtersa** [comment]
Awesome, thanks for the quick turnaround 👍 ! I'll keep an eye out for when the new version is released.

**Comment by pglass** [comment]
Hi, I just released 0.0.3: https://pypi.org/project/tox-pip-version/0.0.3/
